config/src/main/java/tech/pegasys/pantheon/config/CliqueConfigOptions.java

```diff
@@ -34,4 +37,15 @@ public long getEpochLength() {
   public int getBlockPeriodSeconds() {
     return cliqueConfigRoot.getInteger("blockperiodseconds", DEFAULT_BLOCK_PERIOD_SECONDS);
   }
+
+  Map<String, Object> asMap() {
+    final ImmutableMap.Builder<String, Object> builder = ImmutableMap.builder();
+    if (cliqueConfigRoot.containsKey("epochlength")) {
+      builder.put("epochLength", getEpochLength());
+    }
+    if (cliqueConfigRoot.containsKey("blockperiodseconds")) {
+      builder.put("blockPeriodSeconds", getBlockPeriodSeconds());
+    }
```

Comment:
Getting pedantic now but I wonder if we should always include this with the default value if none is set?  Would make this a really useful API for what values are actually in effect.

Reply:
All of our existing json files have them, and a default is not called out in the spec.  Consensus specs are a bad place for "this default that we have all agreed to by convention" values, so I'll always report them.  Also, it makes the code cleaner. :)